source/game/kart/KartObjectProxy.cc

```diff
@@ -3,13 +3,17 @@
 #include "game/kart/CollisionGroup.hh"
 #include "game/kart/KartObject.hh"
 #include "game/kart/KartObjectManager.hh"
+#include "game/kart/KartState.hh"
 #include "game/kart/KartSub.hh"
 #include "game/kart/KartSuspension.hh"
 #include "game/kart/KartTire.hh"
 
+#include "game/system/CourseMap.hh"
+#include "game/system/map/MapdataCannonPoint.hh"
```

Comment:
This alphabetization does not match what you have in `KartMove.cc`. Convention so far is to put subdirectories after all root directory includes, so move this after `RaceManager.hh`

Reply:
```cpp
#include "game/kart/KartSuspension.hh"
#include "game/kart/KartTire.hh"

#include "game/system/CourseMap.hh"
#include "game/system/RaceManager.hh"
#include "game/system/map/MapdataCannonPoint.hh"

#include <egg/math/Math.hh>

#include <algorithm>
```
this what you mean?

Reply:
Yep!